### PR TITLE
Centrifuge App: Configure Admins

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

Adds the Admin section to the Configuration page

### Approvals

- [ ] Dev

### Screenshots

image
image

### Impact

Centrifuge App

### CODE REVIEW

File: nft-studio/src/pages/IssuerPool/Configuration/Admins.tsx [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+            .map(([userAddress, permissions]) => ({
+              address: encodeAddress(userAddress),
+              roles: Object.fromEntries(permissions.roles.map((role) => [role, true])) as Admin['roles'],
+            }))
+        : [],
+    }),
+    [poolPermissions]
+  )
+
+  const { execute, isLoading } = useCentrifugeTransaction('Create pool', (cent) => cent.pools.updateRoles, {
```
Comment (offerijns) [inline nft-studio/src/pages/IssuerPool/Configuration/Admins.tsx:50]:
```suggestion
  const { execute, isLoading } = useCentrifugeTransaction('Update admins', (cent) => cent.pools.updateRoles, {
```

**Review by offerijns — COMMENTED** [review]
This works really well!

File: nft-studio/src/pages/IssuerPool/Configuration/Admins.tsx (hunk 1 above)
Comment (onnovisser) [reply nft-studio/src/pages/IssuerPool/Configuration/Admins.tsx:50]: 🙈

**Review by sophialittlejohn — APPROVED** [review]
Nice job 💪🎉

**Pull request merged**